app/lib/services/voip/MediaCallEvents.ts

```
@@ -94,9 +124,7 @@ function handleVoipAcceptSucceededFromNative(data: VoipPayload, adapters: MediaC
 	NativeVoipModule.clearInitialEvents();
 	useCallStore.getState().setNativeAcceptedCallId(data.callId);
 	if (data.host && isVoipIncomingHostCurrentWorkspace(data.host, adapters.getActiveServerUrl)) {
-		mediaSessionInstance.applyRestStateSignals().catch(error => {
-			mediaCallLogger.error(`${TAG} applyRestStateSignals failed:`, error);
-		});
+		void runReconnectAndReplaySignals();
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)